source/process/security.md

```diff
@@ -88,27 +91,166 @@ The following resources are monitoring for information about new security threat
+1. Are all system, application and device password files encrypted using an industry standard encryption algorithm where technically feasible?
+   - Yes
+
+1. Do IT systems including but not limited to servers, routers, switches, firewalls, databases are privileged accounts (e.g., system or security administrator) that communicate directly with the Internet contain any personally identifiable information (PII) such as: social security numbers, credit card numbers, patient health record information, or other confidential records?
+   - Yes
+
+1. Is all sensitive, protected health information (PHI) and personally identifiable information (PII) protected using an industry standard encryption algorithm where technically feasible?
+   - Yes
+
+1. Are information assets classified?
    - Yes.
 
-1. For all IT systems including but not limited to servers, routers, switches, firewalls, databases are privileged accounts (e.g., system or security administrator) logged at all times and reviewed on at least a quarterly basis?
+1. Are security roles and responsibilities of constituents defined and documented in accordance with the organization’s information security policy?
    - Yes.
 
-1. Are passwords prevented from being displayed in clear text during user authentication or in electronic/printed reports?
+1. Is a background screening performed prior to allowing constituent access to Scoped Systems and Data?
    - Yes.
 
-1. Are all system, application and device password files encrypted using an industry standard encryption algorithm where technically feasible?
-   - Yes
+1. Are new hires required to sign any agreements upon hire?
+   - Yes.
+
+1. Is there a disciplinarily process for non-compliance with information security policies?
+   - Yes, disclosure of confidential information or egregious disregard for documented security policies is grounds for termination. 
+
+1. Is there a constituent termination or change of status process?
+   - Yes.
+
+### Access Control
+
+1. Is access to and maintenance of applications, systems, network components (including routers, databases, firewalls, voice communications servers, voice recording servers, voice response units (VRU) etc), operating systems, virtualization components, hypervisors, or other information objects restricted to authorized personnel only?
+   - Yes.
+
+1. Is access to and maintenance of applications, systems, network components (including routers, databases, firewalls, voice communications servers, voice recording servers, voice response units (VRU) etc), operating systems, virtualization components, hypervisors, or other information objects granted based upon need-to-know job function?
+   - Yes.
+
+1. Are unique user IDs required for all IT systems including but not limited to servers, routers, switches, firewalls, databases are privileged accounts (e.g., system or security administrator)?
```

Comment:
Sorry, I'm not sure what this sentence is asking.  Should it read:

"Are unique user IDs required for all IT systems including but not limited to servers, routers, switches, firewalls and databases, and are these privileged accounts (e.g., system or security administrator)?"

Reply:
I actually meant

```
Are unique user IDs required for all user and privileged accounts (e.g., system or security administrator) to access all IT systems including but not limited to servers, routers, switches, firewalls and databases.
```

Let me know if that works?

Reply:
Yep, that works.  I've updated.